src/twisted/web/http_headers.py

```diff
@@ -81,6 +73,10 @@ class Headers:
         b"x-xss-protection": b"X-XSS-Protection",
```

Comment:
Hmm. Slightly tangential, but it strikes me that `_caseMappings` should probably have a `ClassVar[Mapping[...]]` annotation here, we don't ever want to mutate this one.